tests/bsim/bluetooth/host/l2cap/credits/src/main.c

```diff
@@ -23,7 +23,8 @@ CREATE_FLAG(flag_l2cap_connected);
 #define L2CAP_MTU       (2 * SDU_LEN)
 
 /* Only one SDU transmitted or received at a time */
-NET_BUF_POOL_DEFINE(sdu_pool, 1, L2CAP_MTU, CONFIG_BT_CONN_TX_USER_DATA_SIZE, NULL);
+NET_BUF_POOL_DEFINE(sdu_pool, 1, L2CAP_MTU + BT_L2CAP_SDU_HDR_SIZE,
```

Comment:
```suggestion
NET_BUF_POOL_DEFINE(sdu_pool, 1, BT_L2CAP_SDU_BUF_SIZE(L2CAP_MTU),
```

Reply:
Does it? 

`CONFIG_BT_L2CAP_TX_MTU` is usually used with `BT_L2CAP_SDU_BUF_SIZE`, so that is without the header, right? 

`L2CAP_MTU` is a multiple of `SDU_LEN` which is a multiple of `L2CAP_MPS` which is defined as `CONFIG_BT_L2CAP_TX_MTU`

Given that they are multiples, and not using a constant addition to the size, I would assume that none of the above are including the header size (granted, due the multiples, they can certainly fit it, but none of them are adding `BT_L2CAP_HDR_SIZE`). 

If the intention is that `L2CAP_MTU` contains the size of the header, then I feel the above macro definitions are not clear about that (at least not to me). That being said, it is also unclear to me why we are multiplying by the MTU by 4 (2 * 2) in the first place, so I might just be missing something here.

Reply:
Yes, you're right my bad. MTU & MPS both don't include headers. Updated.